PROJECT-BRIEF.md

```diff
@@ -1,59 +1,186 @@
-# Project Brief, The Collab Lab
+# Project Brief
 
-## Smart Shopping List
+This document serves as your comprehensive guide to all available resources necessary during your time at The Collab Lab. Each week you'll see an agenda that details your pairs for the week and the development work you'll be completing, learning modules to complete, and meetings to attend.
 
-### Locations for things
+## Smart Shopping List Resources
 
 - Site: {PRODUCTION_URL}
 - Repo: {REPO_URL}
 - Clone URL: {REPO_CLONE_URL}
 - Issue list: {PROJECT_BOARD_URL}
 - Database: {CONSOLE_URL}
 
-### Project cadence & duration
+## Program Agenda & Curriculum
 
-The project is expected to take 40 hours per participant, spread over 8 weeks.
-
-Each week, the team of 4 developers will split into 2 pairs of 2 developers each. Each pair will take on a story or task from the issues list to work on together that week.
-
-Pairings will go as follows:
-
-#### Week 1, {DATES_WEEK_1}
-
-1. {DEV_1} & {DEV_2}
-2. {DEV_3} & {DEV_4}
-
-#### Week 2, {DATES_WEEK_2}
-
-1. {DEV_2} & {DEV_3}
-2. {DEV_1} & {DEV_4}
-
-#### Week 3, {DATES_WEEK_3}
-
-1. {DEV_3} & {DEV_1}
-2. {DEV_4} & {DEV_2}
-
-#### Week 4, {DATES_WEEK_4}
-
-1. {DEV_1} & {DEV_2}
-2. {DEV_3} & {DEV_4}
-
-#### Week 5, {DATES_WEEK_5}
-
-1. {DEV_2} & {DEV_3}
-2. {DEV_1} & {DEV_4}
-
-#### Week 6, {DATES_WEEK_6}
-
-1. {DEV_3} & {DEV_1}
-2. {DEV_4} & {DEV_2}
-
-#### Week 7, {DATES_WEEK_7}
-
-1. {DEV_1} & {DEV_2}
-2. {DEV_3} & {DEV_4}
-
-#### Week 8, {DATES_WEEK_8}
-
-1. {DEV_2} & {DEV_3}
-2. {DEV_1} & {DEV_4}
+### Week 1, {DATES_WEEK_1}
+
+| 🤝 Developer pairings | 🖥 Assigned Github issue                                                                                                                    |
+| --------------------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
+| {DEV_1} & {DEV_2}     | [2. As a user, I want to read all of my shopping lists and the items in them]({REPO_URL}/issues/2)                                          |
+| {DEV_3} & {DEV_4}     | [3. As a user, I want to be able to navigate to all the pages in the application by clicking the links in the nav bar]({REPO_URL}/issues/3) |
+
+| 📖 Learning modules                                                                                                                                                                 | Description |
```

Comment:
Also, should we link the surveys in this section as well? I know we'll be sending them via Slack, but it might be a nice way to consolidate all of our links.

Reply:
I like the idea. I'm not totally sure where to post them to make it clear it should be done at the end of the week. Do we post it on the next week's Weekly Sync? 🤔 If you can find a place that makes sense, feel empowered to include them! 👍